cogs/public.py

```diff
@@ -78,7 +78,31 @@ async def serverinfo(self, ctx: commands.Context):
         """
         Present server information.
         """
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        embed = discord.Embed(title=f"{ctx.guild.name} ({ctx.guild.id})", color=discord.Color.green())
+        creation_date = ctx.guild.created_at
+        # unix timestamp
+        timestamp = int(creation_date.timestamp())
+        embed.add_field(name="**Creation Date**", value=f"<t:{timestamp}> (<t:{timestamp}:R>)", inline=False)
+        embed.add_field(name="**Member Count**", value=f"{ctx.guild.member_count}")
+        embed.add_field(name="**Verification Level**", value=f"{ctx.guild.verification_level}")
+        embed.add_field(name="**Owner**", value=f"{ctx.guild.owner}")
+
+        txt = len(ctx.guild.text_channels)
+        vc = len(ctx.guild.voice_channels)
+        ctg = len(ctx.guild.categories)
+        stg = len(ctx.guild.stage_channels)
+        frm = len(ctx.guild.forums)
+        total = txt + vc + ctg + stg + frm
+        embed.add_field(name="**Channels**", value=f"{txt} text, {vc} voice, {ctg} categories, {stg} stage, {frm} forum. {total} total", inline=False)
```

Comment:
Punctuation - `"[...] {total} total."`

Reply:
Hi, sorry I was on a grad trip for the last week. Requested changes arriving soon